.github/workflows/pr.yml

```diff
@@ -52,25 +52,6 @@ jobs:
       uses: actions/setup-go@v2
       with:
         go-version: 1.14.x
-    - name: Restore cached binaries
-      id: cache
-      uses: actions/cache@v2
-      with:
-        path: ~/go/bin
-        key: ${{ runner.os }}-go-bin-${{ hashFiles('**/go.sum') }}
-        restore-keys: |
-          ${{ runner.os }}-go-bin-
-    - name: Restore cached modules dependencies
-      uses: actions/cache@v2
-      with:
-        path: ~/go/pkg/mod
-        key: ${{ runner.os }}-go-mod-${{ hashFiles('**/go.sum') }}
-        restore-keys: |
-          ${{ runner.os }}-go-mod-
-    - name: Install dependencies
-      if: steps.cache.outputs.cache-hit != 'true'
-      run: make dependencies
-      shell: bash
```

Comment:
Yes this change tied in with some of the other changes relating to the way `tfplugindocs` was being installed. I think previously the makefile was installing the package to `GOROOT` (i.e. `~/go/pkg` and `~/go/bin`), which would explain the steps in the workflow caching these directories. 

However when adapting the upstream docs generation changes I found that the output of the `tfplugindocs` tool was quite volatile, given its alpha status, and thought it made sense to use the `vendor` style dependency management to pin in the exact version of the source code used to generate the docs and reduce disparity between all of the contributor's versions of it. In a similar vein I changed it to install the binary to a project-local `./bin` directory to isolate it in case other terraform plugin projects were using different versions of it on anyone's machine. (I think you also made some very similar changes in parallel, e.g. adding a `tools.go` to allow `go mod` to find it, so I ended up merging my tweaks into that as well).

Anyway long story short, with the source code being installed to `./vendor` and the compiled `tfplugindocs` binary being installed to `./bin`, I didn't see a need to manually cache the external global `GOROOT` directories as the GitHub workflow does. (Also it had stopped working for me when I removed the `make dependencies` target, so I had to fix it somehow! 😉)

Does this make sense do you think? Happy to jump on a call and discuss it further if you think that would be helpful

Reply:
No that's fine. Thanks for the background information 👍🏻